botfront/imports/api/project/project.methods.js

```diff
@@ -393,5 +394,17 @@ if (Meteor.isServer) {
             check(projectId, String);
             return Projects.findOne({ _id: projectId }, { fields: { deploymentEnvironments: 1 } });
         },
+
+        async 'project.getLogo'(projectId) {
+            check(projectId, String);
+            const settings = GlobalSettings.findOne({}, {
+                fields: { 'settings.public.logoUrl': 1, 'settings.public.smallLogoUrl': 1 },
+            });
+            const project = await Projects.findOne({ _id: projectId }, { fields: { logoUrl: 1, smallLogoUrl: 1 } });
+            return {
+                logoUrl: project?.logoUrl || settings?.settings?.public?.logoUrl,
+                smallLogoUrl: project?.smallLogoUrl || settings?.settings?.public?.smallLogoUrl,
+            };
+        },
     });
```

Comment:
Didn't we say we'd not add meteor methods anymore?
Not a blocker but we need to decide how we want to handle new client/server communication

Reply:
I did it the same way as the other method used for that page, but I agree we should have a real discussion about which way to keep.